tests/docs-screenshots.spec.ts

```diff
@@ -91,20 +111,617 @@ test("instances", async ({ page }) => {
+    .click();
+  await page.getByTestId("notification-close-button").click();
+
+  await visitInstance(page, vminstance);
+  await page.waitForSelector("text=General");
+  await page.getByTestId("tab-link-Console").click();
+  await page.waitForTimeout(35000);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/desktop_console.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  // Open File Explorer and navigate to /Documents
+  await page.mouse.click(300, 280, { delay: 1000 });
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowRight");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+
+  // Creates new folder
+  await page.mouse.click(725, 525, { button: "right", delay: 1000 });
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowUp");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+  await page.keyboard.type("hello_world");
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/hello_world_desktop.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await forceStopInstance(page, vminstance);
+  await deleteInstance(page, vminstance);
+});
+
+test("LXD - UI Folder - Project", async ({ page }) => {
+  page.setViewportSize({ width: 1440, height: 1000 });
+  await gotoURL(page, "/ui/");
+
+  // Project Screenshots
+  await page.waitForTimeout(1000);
+  await page.getByRole("button", { name: "default", exact: true }).click();
+  await page.getByRole("button", { name: "Create project" }).click();
+  await page.getByPlaceholder("Enter name").fill("my-project");
+  await page
+    .getByRole("combobox", { name: "Features" })
+    .selectOption("customised");
+  await page.getByRole("combobox", { name: "Features" }).focus();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/create_project.png",
+    clip: getClipPosition(240, 0, 1440, 960),
+  });
+
+  await page.getByPlaceholder("Enter name").fill("my-restricted-project");
+  await page.getByText("Allow custom restrictions on").click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/create_restr_project1.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+
+  await page
+    .getByRole("navigation", { name: "Project form navigation" })
+    .getByText("Instances")
+    .click();
+  await setOption(page, "Snapshot creation", "allow");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/create_restr_project2.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+});
```

Comment:
In my test run the storage pools and networks were still loading, maybe add something before these two to ensure both are finished loading.